uamqp/async_ops/client_async.py

```diff
@@ -43,8 +43,6 @@ class AMQPClientAsync(client.AMQPClient):
     :param client_name: The name for the client, also known as the Container ID.
      If no name is provided, a random GUID will be used.
     :type client_name: str or bytes
-    :param loop: A user specified event loop.
-    :type loop: ~asycnio.AbstractEventLoop
```

Comment:
I'm wondering instead of removing the docstring for the loop, should we keep this but adding something like "loop is deprecated in Python >= 3.10"?

my concern is that after removing the docstring of this parameter, the docstring of parameters will be inconsistent with the method signature which leads to intellisense/type hint errors in different IDEs.

Reply:
no type hint errors in both Pycharm and VSCode.